src/runtime/analytics-service-test.js

```diff
@@ -246,6 +253,42 @@ describes.realWin('AnalyticsService', {}, env => {
       await analyticsService.getLoggingPromise();
       expect(loggedErrors.length).to.equal(0);
     });
+
+    describe('SwG Clearcut Service Experiment', () => {
+      /**
+       * @param {number} expectedEventCount
+       */
+      async function verifyExpectedEventCount(expectedEventCount) {
+        // This triggers an event.
+        eventManagerCallback({
+          eventType: AnalyticsEvent.UNKNOWN,
+          eventOriginator: EventOriginator.UNKNOWN_CLIENT,
+          isFromUserAction: null,
+          additionalParameters: null,
+        });
+
+        // These wait for analytics server to be ready to send data.
+        expect(analyticsService.lastAction_).to.not.be.null;
+        await analyticsService.lastAction_;
+        await activityIframePort.whenReady();
+
+        expectOpenIframe = true;
+        expect(eventsLoggedToService.length).to.equal(expectedEventCount);
+      }
+
+      beforeEach(() => {
+        // This ensure nothing gets sent to the server.
+        sandbox.stub(activityIframePort, 'execute').callsFake(() => {});
+      });
+
+      afterEach(async () => {});
+      it('should not log to clearcut if experiment off', async () =>
```

Comment:
https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions
> By default the body is a block statement (zero or more statements surrounded by curly braces). The body may also be an implicitly returned single expression if either: the program logic requires returning a value, or the void operator precedes a single function or method call (using void ensures undefined is returned, prevents leaking values, and communicates intent).

Reply:
Could you also add a line between the two `it`s?